yarn.lock

```diff
@@ -19,7 +19,7 @@
 
 "@babel/code-frame@^7.22.13":
   version "7.22.13"
-  resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.22.13.tgz#e3c1c099402598483b7a8c46a721d1038803755e"
+  resolved "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.22.13.tgz"
```

Comment:
You should probably revert this and re-generate using yarn.

Reply:
Thank you, updated